financial-connections/src/main/java/com/stripe/android/financialconnections/features/institutionpicker/InstitutionPickerScreen.kt

```diff
@@ -127,13 +124,8 @@ private fun InstitutionPickerContent(
     ) {
         when (payload) {
             is Uninitialized,
-            is Loading -> FullScreenGenericLoading()
-
-            is Fail -> UnclassifiedErrorContent(
-                error = payload.error,
-                onCloseFromErrorClick = onCloseFromErrorClick
-            )
-
+            is Loading,
+            is Fail -> FullScreenGenericLoading()
```

Comment:
Why is this not showing a failure screen anymore?

Reply:
it's been moved to a full-screen error via navigation: https://github.com/stripe/stripe-android/blob/a4b8600ffb086e4fafa260cea637643eadff587f/financial-connections/src/main/java/com/stripe/android/financialconnections/features/institutionpicker/InstitutionPickerViewModel.kt#L112

Ideally this confusion should go away by getting rid of asyncs, we can discuss how to make it better short and long term